addons/godot-next/references/inflector.gd

```diff
@@ -0,0 +1,269 @@
+		vocabulary.add_singular("(quiz)zes$", "$1")
+		vocabulary.add_singular("(buz|blit|walt)zes$", "$1z")
+		vocabulary.add_singular("(alumn|alg|larv|vertebr)ae$", "$1a")
+		vocabulary.add_singular("(criteri|phenomen)a$", "$1on")
+		vocabulary.add_singular("([b|r|c]ook|room|smooth)ies$", "$1ie")
+		
+		vocabulary.add_irregular("person", "people")
+		vocabulary.add_irregular("man", "men")
+		vocabulary.add_irregular("human", "humans")
+		vocabulary.add_irregular("child", "children")
+		vocabulary.add_irregular("sex", "sexes")
+		vocabulary.add_irregular("move", "moves")
+		vocabulary.add_irregular("goose", "geese")
+		vocabulary.add_irregular("wave", "waves")
+		vocabulary.add_irregular("die", "dice")
+		vocabulary.add_irregular("foot", "feet")
+		vocabulary.add_irregular("tooth", "teeth")
+		vocabulary.add_irregular("curriculum", "curricula")
+		vocabulary.add_irregular("database", "databases")
+		vocabulary.add_irregular("zombie", "zombies")
+		vocabulary.add_irregular("personnel", "personnel")
+		
+		vocabulary.add_irregular("is", "are", false)
+		vocabulary.add_irregular("that", "those", false)
+		vocabulary.add_irregular("this", "these", false)
+		vocabulary.add_irregular("bus", "buses", false)
+		vocabulary.add_irregular("staff", "staff", false)
+		
+		vocabulary.add_uncountable("equipment")
+		vocabulary.add_uncountable("information")
+		vocabulary.add_uncountable("corn")
+		vocabulary.add_uncountable("milk")
+		vocabulary.add_uncountable("rice")
+		vocabulary.add_uncountable("money")
+		vocabulary.add_uncountable("species")
+		vocabulary.add_uncountable("series")
+		vocabulary.add_uncountable("fish")
+		vocabulary.add_uncountable("sheep")
+		vocabulary.add_uncountable("deer")
+		vocabulary.add_uncountable("aircraft")
+		vocabulary.add_uncountable("oz")
+		vocabulary.add_uncountable("tsp")
+		vocabulary.add_uncountable("tbsp")
+		vocabulary.add_uncountable("ml")
+		vocabulary.add_uncountable("l")
+		vocabulary.add_uncountable("water")
+		vocabulary.add_uncountable("waters")
+		vocabulary.add_uncountable("semen")
+		vocabulary.add_uncountable("sperm")
+		vocabulary.add_uncountable("bison")
+		vocabulary.add_uncountable("grass")
+		vocabulary.add_uncountable("hair")
+		vocabulary.add_uncountable("mud")
+		vocabulary.add_uncountable("elk")
+		vocabulary.add_uncountable("luggage")
+		vocabulary.add_uncountable("moose")
+		vocabulary.add_uncountable("offspring")
+		vocabulary.add_uncountable("salmon")
+		vocabulary.add_uncountable("shrimp")
+		vocabulary.add_uncountable("someone")
+		vocabulary.add_uncountable("swine")
+		vocabulary.add_uncountable("trout")
+		vocabulary.add_uncountable("tuna")
+		vocabulary.add_uncountable("corps")
+		vocabulary.add_uncountable("scissors")
+		vocabulary.add_uncountable("means")
+		vocabulary.add_uncountable("mail")
+		return vocabulary
+
+##### PROPERTIES #####
+
+var _vocabulary: Vocabulary setget, get_vocabulary
+
+##### NOTIFICATIONS #####
+
+func _init(p_vocabulary = Vocabulary.new()):
```

Comment:
Why not have the default value be `null`, and if the value is null in the `_init()` body, just initialize to the `build_default_vocabulary()` return value?